### PR TITLE
Feature/tests for sensitivity

### PR DESCRIPTION
**Pull request by owen-jones-diffblue**
Work in progress - not ready to be merged

### CODE REVIEW

File: regression/goto-analyzer/two-value-int/two-value-int.c [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#include <assert.h>
+
+int main(){
+  int x = 0;
+  assert(x == 0);
+  assert(x == 1);
+  return 0;
+}
+
```
Comment (thk123) [inline regression/goto-analyzer/two-value-int/two-value-int.c]: Please remove blank line.

File: regression/goto-analyzer/two-value-int/two-value-int.c [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#include <assert.h>
+
+int main(){
+  int x = 0;
+  assert(x == 0);
```
Comment (thk123) [inline regression/goto-analyzer/two-value-int/two-value-int.c]: Remove brackets around `==`

File: regression/goto-analyzer/two-value-int/two-value-int.c [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#include <assert.h>
+
+int main(){
```
Comment (thk123) [inline regression/goto-analyzer/two-value-int/two-value-int.c]: Braces on next line please.

**Comment by martin-cs** [comment]
Looks good so far.  Could you add a comment or so to each one saying what is is supposed to test?  Ideas for more: a constant array and access to different locations of the same array (constant and non-constant).

**Comment by owen-jones-diffblue** [comment]
This is now ready for review

**Comment by tautschnig** [comment]
Can the tests be suitably squashed such that at each commit the (overall) test suite would actually succeed?

**Comment by owen-jones-diffblue** [comment]
I've squashed everything into one commit. I do not think any useful information has been lost by doing this.

**Pull request merged**